src/modules/ie-objects/services/ie-objects.service.ts

```diff
@@ -275,6 +284,23 @@ export class IeObjectsService {
 		return allAdapted;
 	}
 
+	public async findObjectsForSitemap(licenses: IeObjectLicense[]): Promise<IeObjectsSitemap[]> {
+		try {
+			const { object_ie: objects } = await this.dataService.execute<
+				FindObjectsForSitemapQuery,
+				FindObjectsForSitemapQueryVariables
+			>(FindObjectsForSitemapDocument, { licenses });
```

Comment:
this will get all 1.4 million objects at once from the database.
I think we better do this like:
```typescript
findObjectsForSitemap(licenses: IeObjectLicense[], offset: number, limit: number): Promise<IPagination<IeObjectsSitemap>> {
   ...
}

const result = findObjectsForSitemap(..., 0, 0);
for (let i = 0; i < result.total; i+=XML_OBJECTS_SIZE) {
	const objects = await findObjectsForSitemap(..., i, i+XML_OBJECTS_SIZE);

	const xmlSitemapString = convertToXml(objects.items);
	await uploadFile(...xmlSitemapString...)
}
```

So we handle 50000 objects at a time, and dont overload the database with one giant request.